includes/Modules/Analytics/Advanced_Tracking/Measurement_Events/Measurement_Event_List.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\Measurement_Event_List
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events;
+
+/**
+ * Class for containing a specific plugin's tracking event list.
+ *
+ * @since n.e.x.t.
+ * @access private
+ * @ignore
+ */
+class Measurement_Event_List {
+
+	/**
+	 * Container for list of events - intended to be used per plugin.
+	 *
+	 * @since n.e.x.t.
+	 * @var Measurement_Event[]
+	 */
+	private $measurement_events = array();
+
+	/**
+	 * Adds a measurement event to the measurement events array.
+	 *
+	 * @since n.e.x.t.
+	 *
+	 * @param Measurement_Event $e The measurement event object.
+	 */
+	protected function add_event( Measurement_Event $e ) {
+		array_push( $this->measurement_events, $e );
```

Comment:
Better to use a proper variable name here, not `$e`. The only place where an abbreviated variable name like this is okay is e.g. for an `Exception`.